src/engine/enginebuffer.cpp

```diff
@@ -551,7 +553,9 @@ void EngineBuffer::slotTrackLoaded(TrackPointer pTrack,
     TrackPointer pOldTrack = m_pCurrentTrack;
     m_pause.lock();
 
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
```

Comment:
Let's switch these code blocks, i.e. the Qt 6 variant should take precedence and be placed before the legacy workaround. Consistency, makes it easier to throw away all legacy code later.

Reply:
Done.